news/2020-09-13-project-loved-september-2020.md

```diff
@@ -0,0 +1,245 @@
+
+Creeper? Aww man... Besides the meme, here we brought to you a beatmap of a world-famous song's remix, *Revenge (ReeK's "Dude, What's A Genre?" Remix)*, mapped by yuzu__rinrin & KTYN. As the original song features a bunch of genres mixed up, this beatmap challenges you with osu!taiko's representative pattern styles, like 1/6 tech, big burst, constant high speed, offbeat and SV changes.\
+Even with those tough pattern style variations, thanks to those brilliant mappers they're well balanced, very fun and fair enough to give you a great experience of a unique genre switching map.
+
+[![](/wiki/shared/news/2020-09-13-project-loved-september-2020/taiko/4-dream-vandalism.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[katagiri - Dream Vandalism](https://osu.ppy.sh/beatmapsets/895400#taiko) by [applerss](https://osu.ppy.sh/users/983349)\
+151 BPM, 2:41 | 6.60★, 8.23★\
+*description written by [Raphalge](https://osu.ppy.sh/users/3918650)*
+
+Do you dream of being a speed player? Walking down the street and hearing people whisper "Hey... Isn't that the guy who smacks his keyboard at over 300 beats per minute?" Well then it's time to prove your worth or have your dreams vandalized by the speed demon himself, applerss!\
+With his mapset of *katagiri - Dream Vandalism*, applerss offers a challenge to the best of the best with 2 difficulties crammed full of 340 BPM carnage. One at 8.23\* for the big boys who want some 1/6 spice as well as a 6.6\* difficulty for those with a slightly more "normal" palette.
+
+## <a id="catch"></a>osu!catch
+
+osu!catch Loved candidates were chosen by [Sartan](https://osu.ppy.sh/users/4100941), [Tenshichan](https://osu.ppy.sh/users/1101600), [Wesley](https://osu.ppy.sh/users/2407265), [Yumeno Himiko](https://osu.ppy.sh/users/1806962) and [Spectator](https://osu.ppy.sh/users/702598)!
+
+---
+
+[![](/wiki/shared/news/2020-09-13-project-loved-september-2020/catch/1-feelin-sky-camellia-s-200step-self-remix.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Camellia - Feelin Sky (Camellia's "200step" Self-remix)](https://osu.ppy.sh/beatmapsets/867338#fruits) by [Zyzyx](https://osu.ppy.sh/users/2888013)\
+200 BPM, 6:10 | 6.80★\
+*description written by [Yumeno Himiko](https://osu.ppy.sh/users/1806962)*
+
+The map chosen for this month is Feelin Sky (Camellia's "200step" Self-remix), well done and popularly known by the public because of the incredible artist Camellia. The mapper Zyzyx is not a regular mapper, his maps are full of creativity and imagination. Feelin Sky is no exception, the map uses a lot of straight patterns combined with hyperdashes, and you can see a lot of sliders having internal hypers which are not quite popular by year 2018. The wub sections are expressed with different streams and high density horizontal movement, it could be challenging for players. In conclusion, I think this is a glorious map and it deserves to have a spot in loved section.
```

Comment:
```suggestion
The map chosen for this month is Feelin Sky (Camellia's "200step" Self-remix), well done and popularly known by the public because of the incredible artist Camellia. 

Zyzyx is no mere regular mapper, as his maps are full of creativity and imagination. Feelin Sky is no exception, the map uses a lot of straight patterns combined with hyperdashes, and you can see a lot of sliders sporting internal hypers which were well ahead of their time. The wub sections are expressed with different streams and high density horizontal movement, posing a challenge for those who dare to try. In conclusion, I think this is a glorious map and it deserves to have a spot in Loved.
```